relational_operators/WorkOrder.proto

```diff
@@ -120,14 +122,15 @@ message HashJoinWorkOrder {
     repeated int32 join_key_attributes = 163;
     optional bool any_join_key_attributes_nullable = 164;
     optional int32 insert_destination_index = 165;
-    optional uint32 join_hash_table_index = 166;
-    optional int32 selection_index = 167;
-    optional fixed64 block_id = 168;
+    optional uint32 join_hash_table_group_index = 166;
+    optional uint32 join_hash_table_index = 167;
```

Comment:
Q: what's the difference between `join_hash_table_group_index` and `join_hash_table_index`.